lib/routes/refund/pages/get_refund/widgets/refund_item_card/refund_item_card.dart

```diff
@@ -23,25 +23,38 @@ class RefundItemCard extends StatelessWidget {
     final ThemeData themeData = Theme.of(context);
 
     final String lastRefundTxId = refundableSwap.lastRefundTxId ?? '';
+    final String txId = refundableSwap.lastRefundTxId ?? refundableSwap.swapAddress;
 
     return Card(
-      color: themeData.customData.paymentListBgColorLight,
+      color: themeData.customData.surfaceBgColor,
       margin: const EdgeInsets.all(16),
       child: Padding(
         padding: const EdgeInsets.all(16.0),
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: <Widget>[
-            const SizedBox(height: 8),
-            if (refundableSwap.swapAddress.isNotEmpty) ...<Widget>[
-              RefundItemCardOriginalTx(swapAddress: refundableSwap.swapAddress),
-              const Divider(
-                height: 32.0,
-                color: Color.fromRGBO(40, 59, 74, 0.5),
-                indent: 0.0,
-                endIndent: 0.0,
+            ShareablePaymentRow(
+              isExpanded: true,
+              tilePadding: EdgeInsets.zero,
+              dividerColor: Colors.transparent,
+              title: 'Transaction',
+              titleTextStyle: themeData.primaryTextTheme.headlineMedium?.copyWith(
+                fontSize: 18.0,
+                color: Colors.white,
               ),
-            ],
+              sharedValue: txId,
+              isURL: true,
+              urlValue: BlockchainExplorerService.formatTransactionUrl(
```

Comment:
If this is a swap address, then there needs to be a different URL used e.g. `formatAddressUrl()`

Reply:
Noted.
These are not part of this PR's scope and was momentarily shown while rebasing.